jwst/cube_build/src/cube_match_sky_pointcloud.c

```diff
@@ -452,306 +485,303 @@ int match_point_msm(double *xc, double *yc, double *zc,
     return 0;
 }
 
-
-PyArrayObject * ensure_array(PyObject *obj, int *is_copy) {
-    if (PyArray_CheckExact(obj) &&
-        PyArray_IS_C_CONTIGUOUS((PyArrayObject *) obj) &&
-        PyArray_TYPE((PyArrayObject *) obj) == NPY_DOUBLE) {
+PyArrayObject *ensure_array(PyObject *obj, int *is_copy)
+{
+    if (PyArray_CheckExact(obj) && PyArray_IS_C_CONTIGUOUS((PyArrayObject *)obj) &&
+        PyArray_TYPE((PyArrayObject *)obj) == NPY_DOUBLE)
+    {
         *is_copy = 0;
-        return (PyArrayObject *) obj;
-    } else {
+        return (PyArrayObject *)obj;
+    }
+    else
+    {
         *is_copy = 1;
-        return (PyArrayObject *) PyArray_FromAny(
-            obj, PyArray_DescrFromType(NPY_DOUBLE), 0, 0,
-            NPY_ARRAY_CARRAY | NPY_ARRAY_FORCECAST, NULL
-        );
+        return (PyArrayObject *)PyArray_FromAny(obj, PyArray_DescrFromType(NPY_DOUBLE), 0, 0,
+                                                NPY_ARRAY_CARRAY | NPY_ARRAY_FORCECAST, NULL);
     }
 }
 
 // Wrapper code that is called from python code and sets up interface with C code.
-static PyObject *cube_wrapper(PyObject *module, PyObject *args) {
-  PyObject *result = NULL, *xco, *yco, *zco, *fluxo, *erro, *coord1o, *coord2o, *waveo, *slicenoo; // codespell:ignore erro
-  PyObject *rois_pixelo, *roiw_pixelo, *scalerad_pixelo, *zcdelt3o, *softrad_pixelo, *weight_pixelo;
+static PyObject *cube_wrapper(PyObject *module, PyObject *args)
+{
+    PyObject *result = NULL, *xco, *yco, *zco, *fluxo, *error, *coord1o, *coord2o, *waveo,
```

Comment:
This pre-commit is too bold and also didn't ask before applying all the changes...

Reply:
I think we need https://github.com/spacetelescope/jwst/pull/9791